tesla_ble_mqtt/CHANGELOG.md

```diff
@@ -22,7 +22,6 @@ On/off and Open/Close entities have been grouped under switches and covers.</p>
    | trunk-open        | trunk (open/close)       |
    | trunk-close       | "                        |
 
```

Comment:
Do you plan to use the auto changelog feature and capture the one from core?
Otherwise you'll need to copy/paste there maybe...

Reply:
There's a bunch of conflicts in that PR .... will need to recreate it from scratch, most likely less work than fix a bunch of files.  I'm a bit busy til August 8th but I'll try to do it if I get spare time.